lapack/tpls/KokkosLapack_trtri_tpl_spec_decl.hpp

```diff
@@ -61,136 +61,141 @@ namespace Impl {
 
 // Handle type and space permutations
-#define KOKKOSBLAS_DTRTRI_BLAS(LAYOUTA, ETI_SPEC_AVAIL)                   \
-  KOKKOSBLAS_TRTRI_BLAS_HOST(double, double, LAYOUTA, Kokkos::HostSpace,  \
-                             ETI_SPEC_AVAIL)                              \
-  KOKKOSBLAS_TRTRI_BLAS_MAGMA(double, magmaDouble_ptr, magma_dtrtri_gpu,  \
-                              LAYOUTA, Kokkos::CudaSpace, ETI_SPEC_AVAIL) \
-  KOKKOSBLAS_TRTRI_BLAS_MAGMA(double, magmaDouble_ptr, magma_dtrtri_gpu,  \
-                              LAYOUTA, Kokkos::CudaUVMSpace, ETI_SPEC_AVAIL)
-
-#define KOKKOSBLAS_STRTRI_BLAS(LAYOUTA, ETI_SPEC_AVAIL)                   \
-  KOKKOSBLAS_TRTRI_BLAS_HOST(float, float, LAYOUTA, Kokkos::HostSpace,    \
-                             ETI_SPEC_AVAIL)                              \
-  KOKKOSBLAS_TRTRI_BLAS_MAGMA(float, magmaFloat_ptr, magma_strtri_gpu,    \
-                              LAYOUTA, Kokkos::CudaSpace, ETI_SPEC_AVAIL) \
-  KOKKOSBLAS_TRTRI_BLAS_MAGMA(float, magmaFloat_ptr, magma_strtri_gpu,    \
-                              LAYOUTA, Kokkos::CudaUVMSpace, ETI_SPEC_AVAIL)
-
-#define KOKKOSBLAS_ZTRTRI_BLAS(LAYOUTA, ETI_SPEC_AVAIL)                        \
-  KOKKOSBLAS_TRTRI_BLAS_HOST(Kokkos::complex<double>, std::complex<double>,    \
-                             LAYOUTA, Kokkos::HostSpace, ETI_SPEC_AVAIL)       \
-  KOKKOSBLAS_TRTRI_BLAS_MAGMA(Kokkos::complex<double>, magmaDoubleComplex_ptr, \
-                              magma_ztrtri_gpu, LAYOUTA, Kokkos::CudaSpace,    \
-                              ETI_SPEC_AVAIL)                                  \
-  KOKKOSBLAS_TRTRI_BLAS_MAGMA(Kokkos::complex<double>, magmaDoubleComplex_ptr, \
-                              magma_ztrtri_gpu, LAYOUTA, Kokkos::CudaUVMSpace, \
-                              ETI_SPEC_AVAIL)
-
-#define KOKKOSBLAS_CTRTRI_BLAS(LAYOUTA, ETI_SPEC_AVAIL)                        \
-  KOKKOSBLAS_TRTRI_BLAS_HOST(Kokkos::complex<float>, std::complex<float>,      \
-                             LAYOUTA, Kokkos::HostSpace, ETI_SPEC_AVAIL)       \
-  KOKKOSBLAS_TRTRI_BLAS_MAGMA(Kokkos::complex<float>, magmaFloatComplex_ptr,   \
-                              magma_ctrtri_gpu, LAYOUTA, Kokkos::CudaSpace,    \
-                              ETI_SPEC_AVAIL)                                  \
-  KOKKOSBLAS_TRTRI_BLAS_MAGMA(Kokkos::complex<float>, magmaFloatComplex_ptr,   \
-                              magma_ctrtri_gpu, LAYOUTA, Kokkos::CudaUVMSpace, \
-                              ETI_SPEC_AVAIL)
+#define KOKKOSLAPACK_DTRTRI_LAPACK(LAYOUTA, ETI_SPEC_AVAIL)                   \
+  KOKKOSLAPACK_TRTRI_LAPACK_HOST(double, double, LAYOUTA, Kokkos::HostSpace,  \
+                                 ETI_SPEC_AVAIL)                              \
+  KOKKOSLAPACK_TRTRI_LAPACK_MAGMA(double, magmaDouble_ptr, magma_dtrtri_gpu,  \
+                                  LAYOUTA, Kokkos::CudaSpace, ETI_SPEC_AVAIL) \
```

Comment:
As before, I think we should check `KOKKOS_ENABLE_CUDA` before using `Kokkos::CudaSpace`

Reply:
Thanks. Done.